src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -4359,6 +4359,47 @@ def test_ps_s3_multiple_topics_notification():
     http_server.close()
 
 
+@attr('kafka_test')
+def test_ps_s3_multiple_topics_kafka():
+    """ test multiple topics"""
+
+    conn = connection()
+    bucket_name = gen_bucket_name()
+    topic_name1 = bucket_name + TOPIC_SUFFIX + '1'
+    topic_name2 = bucket_name + TOPIC_SUFFIX + '2'
+    topic_name3 = bucket_name + TOPIC_SUFFIX + '3'
+    # Start kafka receiver
+    task, receiver = create_kafka_receiver_thread(topic_name1)
```

Comment:
see my previos comment. no need to start the kafka reciever - we are not sending any notifications, just configuring topics

Reply:
note that you can still create a topic with a kafka endpoint, we don't check whether the endpoint is there until we send actual topics to it